lib/Installation/Partitioner/LibstorageNG/v4_3/CreatePartitionTablePage.pm

```diff
@@ -27,7 +27,8 @@ sub new {
 
 sub init {
     my $self = shift;
-    $self->{btn_next} = $self->{app}->button({id => 'next'});
+    $self->{rd_partition_type} = $self->{app}->radiobutton({id => 'msdos'});
```

Comment:
From the id looks like it should be called `rb_msdos_partition table`. Or am I missing something?

Reply:
yes, makes sense.